### ISSUE TITLE
UPDATE ORDERS

### ISSUE THREAD

**Issue by keanacobarde** [keanacobarde/HHPWsBe#5]
# User Story

# Acceptance Criteria
- Endpoint URL: "/orders/{orderId}"
As pulled from acceptance criteria highlighted by MVP: 
```
Create and Update Orders: Functionality to add new orders and modify existing ones, including customer details and order type.
```
NOTE: This does not include the editing of the items within an order, only the order details. Adding and deleting of items from an order will be outlined in a separate issue ticket categorized as 'order items'.

# Dependecies
- All setup tickets will need to be completed before work on this issue is started. 

# Dev Notes
_CreekRiver - Campsites_
```
app.MapPut("/api/campsites/{id}", (CreekRiverDbContext db, int id, Campsite campsite) =>
{
    Campsite campsiteToUpdate = db.Campsites.SingleOrDefault(campsite => campsite.Id == id);
    if (campsiteToUpdate == null)
    {
        return Results.NotFound();
    }
    campsiteToUpdate.Nickname = campsite.Nickname;
    campsiteToUpdate.CampsiteTypeId = campsite.CampsiteTypeId;
    campsiteToUpdate.ImageUrl = campsite.ImageUrl;

    db.SaveChanges();
    return Results.NoContent();
});
```
